web-platform-features/deprecated-performance-apis.json

```diff
@@ -0,0 +1,28 @@
+{
+  "identifier": "deprecated-performance-apis",
+  "shortName": "Deprecated Performance APIs",
+  "longName": "Deprecated Web Performance Measurement APIs",
+  "description": "Deprecated Performance APIs that shouldn't be used anymore.",
+  "constituentFeatures": [
+    {
+      "source": "bcd",
+      "query": "api.PerformanceNavigation"
```

Comment:
`api.PerformanceNavigation` contains a few subfeatures. When you wrote this, were you thinking any of the following?

- You don't care about any of the subfeatures, one way or the other?
- Tooling would help you complete the file here, to create a flat list of features?
- Tooling would assume that `api.PerformanceNavigation` means something like `api.PerformanceNavigation + api.PerformanceNavigation.*` ?

Reply:
I think it might make sense to actually spell out all features here. What happens sometimes is that there is going to be
`api.PerformanceNavigation.method.returns_somethingNew` and I guess that should then be taken into a group or it will be taken into a group automatically if we assume `api.PerformanceNavigation.*` (that auto-assigning to groups might work or might not, hard to tell right now).